README.md

```diff
@@ -382,26 +386,25 @@ Email [help@rainforestqa.com](mailto:help@rainforestqa.com) if you're having tro
 1. Update the `version` constant in `rainforest-cli.go` following [semantic versioning](http://semver.org/)
 1. Merge to master
 
+### Releasing
+1. **Docker** Tag `master` after merging: `git tag vX.Y.Z && git push --tags`
+1. **GitHub** Wait for the CircleCI build to finish. This will create a [draft GitHub Release](https://github.com/rainforestapp/rainforest-cli/releases). Edit the description as appropriate and publish the release.
+1. **Homebrew** Update https://github.com/rainforestapp/homebrew-public to use the latest URL and SHA256. Both can be found in the GitHub Release assets. Additionally, the SHA256 is output as part of the CircleCI `Release` job.
+1. **Chocolatey** [Run the workflow here](https://github.com/rainforestapp/rainforest-cli-chocolatey/actions/workflows/chocolatey.yml) to build & release an updated Chocolatey package. Note, this uses the release you published earlier.
 
-## Release:
-1. Tag `master` after merging: `git tag vX.Y.Z && git push --tags`
-1. Wait for the CircleCI build to finish. This will create a [draft GitHub Release](https://github.com/rainforestapp/rainforest-cli/releases). Edit the description as appropriate and publish the release.
-1. Update https://github.com/rainforestapp/homebrew-public to use the latest URL and SHA256. Both can be found in the GitHub Release assets. Additionally, the SHA256 is output as part of the CircleCI `Release` job.
-1. [Run the workflow here](https://github.com/rainforestapp/rainforest-cli-chocolatey/actions/workflows/chocolatey.yml) to build & release an updated Chocolatey package. Note, this uses the release you published earlier.
-
-### Releasing a beta version
+### Releasing a beta version (Docker / GitHub)
 Simply tag a commit with an alpha or beta version.
 ```bash
 git tag vX.Y.Z-alpha.N # or vX.Y.Z-beta.N
 git push origin vX.Y.Z-alpha.N
 ```
 
-## Rollback:
+### Rolling back
 Should you have to rollback, you will need to:
 
-1. Delete the release in question. CLI will 'update' itself to the latest public version, which should downgrade users on the next try
+1. Delete the release in question. The CLI will 'update' itself to the latest public version, which should downgrade users on the next try
```

Comment:
Make this a bit more definitive? Not sure if my suggestion is an improvement.

```suggestion
1. Delete the Github release you want to rollback from. Then the next time the CLI is run it will download the latest available release
```